accelerator/BUILD

```diff
@@ -5,8 +5,18 @@ load("@io_bazel_rules_docker//container:container.bzl", "container_push")
 
 cc_binary(
     name = "accelerator",
-    srcs = ["server.cc"],
-    copts = ["-DLOGGER_ENABLE"] + select({
+    srcs = select({
+        "//connectivity/mqtt:mqtt_enable": [
+            "config.c",
+            "config.h",
+            "conn_mqtt.c",
+        ],
+        "//conditions:default": ["main.c"],
```

Comment:
Do we need `config.c/config.h` when `//conditions:default`?

Reply:
I think we don't need `config.c/config.h` in `//conditions:default` since `main.c` does not directly include `config.h`

Reply:
I think we don't need `config.c/config.h` in `//conditions:default` since `main.c` does not directly include `config.h`

Reply:
I think we don't need `config.c/config.h` in `//conditions:default` since `main.c` does not directly include `config.h`

Reply:
I think we can remove `config.c/config.h` since there's `:`ta_config` in the `deps`